src/test/scala/zio/PrometheusSpec.scala

```diff
@@ -102,7 +102,7 @@ object PrometheusSpec {
     r004 <- write004(someExternalRegistry)
   } yield r004
 
-  val rt = Runtime.unsafeFromLayer(Metrics.live(config) ++ Clock.live ++ Console.live)
+  val rt = Runtime.unsafeFromLayer((Clock.live >+> Metrics.live(config)) ++ Console.live)
```

Comment:
We should make a note here to look at how to refactor this. `unsafeFromLayer` is designed in use for mixed applications where you need to unsafely run effects. I don't think we should need it in a pure ZIO application like this.

Reply:
I replaced the usage of `Runtime.unsafeFromLayer` by providing the metrics layer to the program and then using the default runtime:

```
    val server = Runtime.default.unsafeRun(program.provideSomeLayer[Console with Clock](Metrics.live(config)))
```

The type annotation `Console with Clock` is necessary in order to make the compiler happy. I did not find a way to get rid of the type annotation.